execution_workflows/Aptardi/bin/make_identification_bed.py

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env python
+
+import sys
+
+input_gtf_path=sys.argv[1]
+input_gtf=open(input_gtf_path,"r")
+output_bed=open(input_gtf_path.split(".")[0]+"_identification.bed","w")
```

Comment:
Can we have the option to specify the output file name? i.e. as a command line argument and specified as an option in the main pipeline config file.

Reply:
Yes, I have added a `--output_bed` flag for the user to specify the output bed file name, and if the `--output_bed` is not specified, it will use the default naming convention.